Turn off diurnal flux, and other fixes

fischer-ncar opened the pull request:
Turn off diurnal flux, fix B1850Cw compset, move SMS_D.f19_g16.B1850
to yellowstone, update cori Intel compiler.

Test suite: SMS.f19_f19.FC5CLUBB.corip1_intel
                 SMS.f19_g16.B1850Cw.yellowstone_intel
Test baseline:
Test namelist changes: SMS_D.f19_g16.B1850 moved to yellowstone
Test status: some answer changes

Fixes: 

User interface changes?:

Code review:

fischer-ncar committed:
Merge remote-tracking branch 'upstream/master'

fischer-ncar committed:
Turn off diurnal flux, fix B1850Cw compset, move SMS_D.f19_g16.B1850
to yellowstone, update cori Intel compiler.

Test suite: SMS.f19_f19.FC5CLUBB.corip1_intel
            SMS.f19_g16.B1850Cw.yellowstone_intel
Test baseline:
Test namelist changes: SMS_D.f19_g16.B1850 moved to yellowstone
Test status: some answer changes

Fixes:

User interface changes?:

Code review:

jedwards4b commented on cime_config/cesm/machines/config_machines.xml:
```diff
@@ -601,7 +601,7 @@
       <modules compiler="intel">
 	<command name="load">PrgEnv-intel</command>
-	<command name="switch">intel intel/2016.0.109</command>
+	<command name="switch">intel intel/2016.0.2.181</command>
```
This is wrong - it should be:  intel/16.0.2.181   without the 20

fischer-ncar committed:
Fix Intel module name for corip1, should be 16.0.2.181 instead of 2016.0.2.181

Test suite:
Test baseline:
Test namelist changes:
Test status:

Fixes:

User interface changes?:

Code review:
